docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/yaml-parse.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_label: yaml-parse
+description: Parses a YAML string and extracts values based on specified expressions.
+---
+
+# `yaml-parse`
```

Comment:
I have been thinking some more about this, and it might be easier if parse steps were prefixed with `parse-`. I.e., `parse-yaml`, `parse-json`, etc., what do you think?

Reply:
Given we already have a `yaml-update` step, we should address this at a later moment while simultaneously aliasing that step to e.g. `update-yaml`.